src/Generation/GapicClientV2Generator.php

```diff
@@ -478,15 +478,23 @@ private function defaultTransport()
 
     private function supportedTransports()
     {
-        if ($this->serviceDetails->transportType !== Transport::REST) {
```

Comment:
How can I run a test that uses the gRpc only?

Reply:
We would need to update one of the test clients, like we have here:

https://github.com/googleapis/gapic-generator-php/blob/main/tests/Integration/BUILD.bazel#L126

But use a different client (or add a new one)